NuRadioReco/utilities/fft.py

```diff
@@ -2,30 +2,40 @@
 
 """
 A wrapper around the numpy fft routines to achive a coherent normalization of the fft
-As we have real valued data in the time domain, we use the 'real ffts' that omit the negative frequencies in Fourier 
-space. To account for the missing power in the frequency domain, we multiply the frequency spectrum by sqrt(2), 
-and divide the iFFT with 1/sqrt(2) accordingly. Then, a calculation of the power leads the same result in 
+As we have real valued data in the time domain, we use the 'real ffts' that omit the negative frequencies in Fourier
+space. To account for the missing power in the frequency domain, we multiply the frequency spectrum by sqrt(2),
+and divide the iFFT with 1/sqrt(2) accordingly. The frequency spectrum is divided by the sampling rate so that the
+units if the spectrum are volts/GHz instead of volts/bin and the voltage in the frequency domain is independent of the
+sampling rate.
+Then, a calculation of the power leads the same result in
 the time and frequency domain, i.e.
 np.sum(trace**2) * dt = np.sum(spectrum**2) * df
```

Comment:
this does not hold anymore! Its actually confusing because no we have:
`np.sum(trace**2) * dt = np.sum(spectrum**2) * df**3`
did you make sure that the energy fluence calculation is updated accordingly?

Reply:
The energy fluence is calculated in the time domain, so it's not affected by this change.

Reply:
yes but the doc string is wrong

Reply:
Fixed the docstring